README.md

```diff
@@ -320,9 +320,21 @@ Install Galaxy on your local system with all the default options:
     galaxy_server_dir: /srv/galaxy
   connection: local
   roles:
-     - galaxy
-```
-
+     - galaxyproject.galaxy
+```  
+  
+If your Ansible version >= 2.10.4, then  
+```yaml
+- hosts: localhost
+  vars:
+    galaxy_server_dir: /srv/galaxy
+    ansible_user: 'anyname'
+  connection: local
```

Comment:
We generally avoid encoding those in the playbook, they're part of the connection variables and should be written in the hosts file.

Reply:
@hexylena 
I do understand you.
However:
1) connection: local was there before. I didn't change nor add it.
2) without the second parameter it fails and there is no mention that it should be set. So, we could drop all the cause, but somehow point it out, so a person will save some time and won't come to the chat for help with the already known issue.
It should be maximum user-friendly for newcomers, shouldn't it?

Reply:
Will open a new PR, according to the notes.